starboard/key.h

```diff
@@ -222,6 +222,9 @@ typedef enum SbKey {
 
   kSbKeyInfo = 0x1C9,
   kSbKeyGuide = 0x1CA,
+#if SB_API_VERSION >= SB_RECORD_KEY_CODE_API_VERSION
+  kSbKeyRecord = 0x1CB,
```

Comment:
In the ATSC spec listed above this block, that value is reserved for VK_TELETEXT 

But it has a definition for VK_RECORD with value 416 (0x1A0)

VK_RECORD = 416

http://www.atsc.org/wp-content/uploads/2015/03/a_100_4.pdf

And since the W3 `UI Events KeyboardEvent key Values` Spec in section `3.18. Media Controller Keys`  listed the "MediaRecord" key value with VK_RECORD in the description, that seems like a good value to use.

https://www.w3.org/TR/uievents-key/#keys-media-controller

Reply:
Thanks Jelle!
The proposal makes sense to me. Let's define kSbKeyRecord = 0x1A0 to be consistent with the rest of the keys in that section. And we can added it just before kSbKeyChannelUp = 0x1AB.

Reply:
Updated the value to 0x1A0